samples/server/petstore/java-vertx-web/src/main/java/org/openapitools/vertxweb/server/api/PetApiHandler.java

```diff
@@ -22,10 +22,10 @@
 
     private static final Logger logger = LoggerFactory.getLogger(PetApiHandler.class);
 
-    private final PetApi apiImpl;
+    private final PetApi api;
 
-    public PetApiHandler() {
-        this.apiImpl = new PetApiImpl();
+    public PetApiHandler(PetApi api) {
```

Comment:
should this keep old constructor and the new one? just deprecate the old one. we've got a breaking change otherwise that would slow down merging it

Reply:
I would love to remove the old constructor if possible, in order to avoid coupling the handler to the api implementation, so when one generates the project and want to exclude the "stub api implementations", it can just do it without manually removing these constructors too. Does that makes sense?

Reply:
add label that this PR had breaking change without fallback

Reply:
How can i do it?

Reply:
> I would love to remove the old constructor if possible

What about keeping the old constructor and mark it as `@deprecated` so that we can include this change in the upcoming v5.1.0 release?

Otherwise, it will be included in the next major release v6.0.0, which is scheduled in 2022.

Reply:
oh, if you cant do it then someone merging it will have to do it.

Reply:
2022 is definitely too far, I'll re-add the constructor

Reply:
Default constructor is back